### ISSUE TITLE
Localized numbers in fields are re-displayed in english after an invalid form submit

### ISSUE THREAD

**Issue by berturion** [zendframework/zend-form#23]
Say we have a field containing a float number in french (with a comma as decimal separator). If the user enter `4,5` (a valid value) and submits the form but the form is invalid for another reason, the form is displayed again to the user with the number in english format (with a dot as decimal separator).
Is it normal ?

**Comment by weierophinney** [comment]
@berturion Yes; internally, the values are passed through a normalization filter *first*, prior to validation, and the normalized value is also what is used then to populate errors and form values.

**Comment by berturion** [comment]
Ok, I understand. Is there a way to filter the values again before populating errors and form values in order to have values in a valid localized format? Because the problem is that the user has to correct already valid filled values he entered... It is quiet annoying, not very user friendly...

**Comment by michalbundyra** [comment]
This repository has been closed and moved to laminas/laminas-form; a new issue has been opened at https://github.com/laminas/laminas-form/issues/51.
